examples/adc.rs

```diff
@@ -0,0 +1,78 @@
+//! ADC example
+
+#![no_main]
+#![no_std]
+
+extern crate panic_halt;
+
+use core::fmt::Write;
+use nb::block;
+
+use lpc8xx_hal::{
+    cortex_m_rt::entry,
+    delay::Delay,
+    prelude::*,
+    syscon::clocksource::{AdcClock, UsartClock},
+    Peripherals,
+};
+
+#[entry]
+fn main() -> ! {
+    let p = Peripherals::take().unwrap();
+
+    let mut delay = Delay::new(p.SYST);
+    let swm = p.SWM.split();
+    let mut syscon = p.SYSCON.split();
+
+    #[cfg(feature = "82x")]
```

Comment:
I think we don't need LPC82x support for this. Regressions would be bad, of course, but we don't need feature parity on new features. I don't have time to really get into it right now, and I think it's perfectly fine to leave this until someone (maybe me, later) actually needs it, and invests the time to get it up to par.

(By the way, this won't work anyway, as the example requires `845` in Cargo.toml.)

Reply:
Yeah, wanted ci to pass. But it's obvious enough how to adapt it to lpc82x